rpc/block_create.c

```diff
@@ -1047,7 +1079,28 @@ block_create_cli_1_svc_st(blockCreateCli *blk, struct svc_req *rqstp)
   GB_STRCPYSTATIC(cobj.gbid, gbid);
   GB_STRDUP(cobj.block_hosts,  blk->block_hosts);
 
-  if (blk->blk_size) { // Create V3
+  if (!resultCaps[GB_CREATE_TCMUR_TIMEOUT_CAP]) { // Create V4
```

Comment:
should this be named after GB_CREATE_IO_TIMEOUT_CAP because the cap name is now create_io_timeout, and all other places replace tcmur with io where ever applicable.

Reply:
Done.